HHS/Head-Start-TTADP
# [TTAHUB-1387] Get all search results and add report text filter to regional dashboard

## Description of change

- Add report text filter to regional dashboard.
- Modify search to return all results (needed for widgets on dashboard)

## How to test

- Go to regional dashboard and add a report text filter.
- Make sure the Report text search return all matches.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1387


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
